client/src/lib/components/CaptchaV2.svelte

```diff
@@ -6,20 +6,25 @@
 
 	const captchaId = "captcha_element";
 
-	onMount(() => {
-		const darkTheme =
-			window.matchMedia && window.matchMedia("(prefers-color-scheme: dark)").matches;
-		const mobileScreen = window.innerHeight > window.innerWidth;
+	let componentMounted: boolean;
 
-		if (window.grecaptcha) {
+	onMount(() => {
+		window.captchaLoaded = () => {
+			const darkTheme =
+				window.matchMedia && window.matchMedia("(prefers-color-scheme: dark)").matches;
+			const mobileScreen = window.innerHeight > window.innerWidth;
+
+			if (!window.grecaptcha) {
+				throw new Error("grecaptcha is undefined!");
+			}
```

Comment:
Should we force the browser to reload the page if this happen?

Reply:
I wouldn't go down that road, to avoid infinite reloads.